### PR TITLE
[FR] Add Skinning Contest #1

### PR DESCRIPTION
**Pull request by bxnn**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+---
+tags:
+  - skinning
+  - skinship
+  - skins
+---
+
+# Skinning Contest #1
+
+![Skinning Contest #1 banner](img/banner.jpg)
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md]:
```suggestion
![Bannière du Skinning Contest #1](img/banner.jpg)
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+Le **Skinning Contest #1** était un concours de skinning ouvert à tous organisé par [skinship](https://skinship.xyz), une des plus grandes communautés de skinning d'osu!. C'était le premier concours de skinning officiel, et la première itération des concours de skinning par skinship.
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:12]:
```suggestion
Le **Skinning Contest #1** était un concours de skinning ouvert à tous organisé par [skinship](https://skinship.xyz), une des plus grandes communautés de skinning d'osu!. C'était le premier concours de skinning officiel, et la première itération des Skinning Contests organisés par skinship.
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+## Planning du Concours
+
+| Evenement | Date |
+| --: | :-- |
+| Annonce | 08-03-2021 |
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:18]:
```suggestion
| Annonce | 08/03/2021 |
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+| Phase d'envoi | 08-03-2021/08-04-2021 |
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md]:
```suggestion
| Phase d'envoi | 08/03/2021 - 08/04/2021 |
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+| Phase de vote | 29-04-2021/13-05-2021 |
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md]:
```suggestion
| Phase de vote | 29/04/2021 - 13/05/2021 |
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+| Résultats | 27-05-2021 |
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md]:
```suggestion
| Résultats | 27/05/2021 |
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+
+## Prix
+
+| Position | Prix |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1ère place") | 6 mois d'osu!supporter, un badge de profil unique |
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md]:
```suggestion
| ![Couronne en or](/wiki/shared/crown-gold.png "1ère place") | 6 mois d'osu!supporter, un badge de profil unique |
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+| ![Silver crown](/wiki/shared/crown-silver.png "2nde place") | 4 mois d'osu!supporter, un badge de profil unique |
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md]:
```suggestion
| ![Couronne d'argent](/wiki/shared/crown-silver.png "2nde place") | 4 mois d'osu!supporter, un badge de profil unique |
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3ème place") | 2 mois d'osu!supporter, un badge de profil unique |
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md]:
```suggestion
| ![Couronne de bronze](/wiki/shared/crown-bronze.png "3ème place") | 2 mois d'osu!supporter, un badge de profil unique |
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+  - skins
+---
+
+# Skinning Contest #1
+
+![Skinning Contest #1 banner](img/banner.jpg)
+
+Le **Skinning Contest #1** était un concours de skinning ouvert à tous organisé par [skinship](https://skinship.xyz), une des plus grandes communautés de skinning d'osu!. C'était le premier concours de skinning officiel, et la première itération des concours de skinning par skinship.
+
+## Planning du Concours
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:14]:
```suggestion
## Planning du concours
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+## Organisation
+
+Le Skinning Contest #1 a été tenu par plusieurs membres de la communauté.
+
+| Membre | Rôle |
+| :-- | :-- |
+| ::{ flag=DE }:: [Master-TigerKun](https://osu.ppy.sh/users/10688456) | Planning du concours |
+| ::{ flag=RO }:: [Matt2e2](https://osu.ppy.sh/users/12144912) | Planning du concours, design graphique |
+| ::{ flag=NL }:: [Roan](https://osu.ppy.sh/users/8214639) | Planning du concours |
+| ::{ flag=DE }:: [RockRoller](https://osu.ppy.sh/users/8388854) | Planning du concours, screening des entrées , preparation du showcase |
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:42]:
```suggestion
| ::{ flag=DE }:: [RockRoller](https://osu.ppy.sh/users/8388854) | Planning du concours, screening des entrées, préparation du showcase |
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+| ::{ flag=GB }:: [tetsui](https://osu.ppy.sh/users/10974678) | Planning du concours, preparation du showcase |
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:43]:
```suggestion
| ::{ flag=GB }:: [tetsui](https://osu.ppy.sh/users/10974678) | Planning du concours, préparation du showcase |
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+| ::{ flag=NL }:: [vvivi](https://osu.ppy.sh/users/10432755) | Planning du concours |
+| ::{ flag=PL }:: [watterino](https://osu.ppy.sh/users/3512261) | Monteur vidéo, preparation du showcase |
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:45]:
```suggestion
| ::{ flag=PL }:: [watterino](https://osu.ppy.sh/users/3512261) | Monteur vidéo, préparation du showcase |
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 14]
```diff
@@ -0,0 +1,117 @@
...
+
+## Liens
+
+- **[Page du concours](https://osu.ppy.sh/community/contests/126)**
+- [Article d'annonce](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement)
+- [Article de Vote](https://osu.ppy.sh/home/news/2021-04-29-skinning-contest-voting-open)
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:51]:
```suggestion
- [Article des votes](https://osu.ppy.sh/home/news/2021-04-29-skinning-contest-voting-open)
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 15]
```diff
@@ -0,0 +1,117 @@
...
+- [Article des résultats](https://osu.ppy.sh/home/news/2021-05-27-skinning-contest-results)
+- [Thread de discussion](https://osu.ppy.sh/community/forums/topics/1266342)
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:53]:
```suggestion
- [Fil de discussion](https://osu.ppy.sh/community/forums/topics/1266342)
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 16]
```diff
@@ -0,0 +1,117 @@
...
+- [Thread d'envoi des entrées](https://osu.ppy.sh/community/forums/topics/1266324)
+- [Serveur Discord](https://discord.skinship.xyz)
+- [Site Officiel](https://skinship.xyz)
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:56]:
```suggestion
- [Site officiel](https://skinship.xyz)
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 17]
```diff
@@ -0,0 +1,117 @@
...
+| ::{ flag=PL }:: [watterino](https://osu.ppy.sh/users/3512261) | Monteur vidéo, preparation du showcase |
+
+## Liens
+
+- **[Page du concours](https://osu.ppy.sh/community/contests/126)**
+- [Article d'annonce](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement)
+- [Article de Vote](https://osu.ppy.sh/home/news/2021-04-29-skinning-contest-voting-open)
+- [Article des résultats](https://osu.ppy.sh/home/news/2021-05-27-skinning-contest-results)
+- [Thread de discussion](https://osu.ppy.sh/community/forums/topics/1266342)
+- [Thread d'envoi des entrées](https://osu.ppy.sh/community/forums/topics/1266324)
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:54]:
```suggestion
- [Post forum des entrées au concours](https://osu.ppy.sh/community/forums/topics/1266324)
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 18]
```diff
@@ -0,0 +1,117 @@
...
+
+## Podium
+
+*Pour les resultats complets, se référer à la [page du concours](https://osu.ppy.sh/community/contests/126).*
+
+![](img/podium.png "Podium du Skinning Contest #1")
+
+## Règles
+
+- Les entrées ne doivent pas contenir du contenu inaproprié, malveillant ou épileptique, et doivent suivre les [règles de la communauté osu!](/wiki/Rules).
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:102]:
```suggestion
- Les entrées ne doivent pas contenir de contenu inaproprié, malveillant ou épileptique, et doivent suivre les [règles de la communauté d'osu!](/wiki/Rules).
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 19]
```diff
@@ -0,0 +1,117 @@
...
+- Tous les assets doivent être crées par vous, ou après demande et crédit de l'auteur. Cela inclut, mais ne se limite pas à:
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:103]:
```suggestion
- Tous les assets doivent être créés par vous-même, ou utilisés après demande et crédit de l'auteur. Cela inclut, mais ne se limite pas à :
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 20]
```diff
@@ -0,0 +1,117 @@
...
+  - polices
+  - icônes
+  - textures
+  - oeuvres (ex: oeuvres d'animes)
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:107]:
```suggestion
  - œuvres (ex : œuvres d'animes)
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 21]
```diff
@@ -0,0 +1,117 @@
...
+  - sons
+- La plupart des menus in-game et au moins un mode de jeu doivent être skinnés. En d'autres termes, en plus des élémets de gameplay, les éléments suivants ne doivent pas être laissés par défaut:
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:109]:
```suggestion
- La plupart des menus en jeu et au moins un mode de jeu doivent être skinnés. En d'autres termes, en plus des éléments de gameplay, les éléments suivants ne doivent pas être laissés par défaut :
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 22]
```diff
@@ -0,0 +1,117 @@
...
+  - Panneau de ranking
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:110]:
```suggestion
  - Écran de résultats
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 23]
```diff
@@ -0,0 +1,117 @@
...
+  - Selection de map (pas forcément la sélection de mode)
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:111]:
```suggestion
  - Sélection de beatmap (pas forcément la sélection de mode)
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 24]
```diff
@@ -0,0 +1,117 @@
...
+  - Icônes de mods
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:112]:
```suggestion
  - Icônes des mods
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 25]
```diff
@@ -0,0 +1,117 @@
...
+  - Menus de pause et de fail
+  - Le main menu est optionnel, mais fortement recommandé
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:114]:
```suggestion
  - Le menu principal est optionnel, mais fortement recommandé
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 26]
```diff
@@ -0,0 +1,117 @@
...
+- Les entrées doivent être des nouvelles créations, seuls les skins postés durant la période d'envoi seront acceptés.
+- Chaque entrée doit être accompagnée d'un thread sur le forum pour participer au concours.
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:116]:
```suggestion
- Chaque entrée doit être accompagnée d'un post sur le forum pour participer au concours.
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 27]
```diff
@@ -0,0 +1,117 @@
...
+![Bannière du Skinning Contest #1](img/banner.jpg)
+
+Le **Skinning Contest #1** était un concours de skinning ouvert à tous organisé par [skinship](https://skinship.xyz), une des plus grandes communautés de skinning d'osu!. C'était le premier concours de skinning officiel, et la première itération des concours de skinning par skinship.
+
+## Planning du Concours
+
+| Evenement | Date |
+| --: | :-- |
+| Annonce | 08-03-2021 |
+| Phase d'envoi | 08/03/2021/08/04/2021 |
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:19]:
```suggestion
| Phase d'envoi | 08/03/2021 - 08/04/2021 |
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 28]
```diff
@@ -0,0 +1,117 @@
...
+| Phase de vote | 29/04/2021/13/05/2021 |
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:20]:
```suggestion
| Phase de vote | 29/04/2021 - 13/05/2021 |
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 29]
```diff
@@ -0,0 +1,117 @@
...
+| Résultats | 27/05/2021 |
+
+## Prix
+
+| Position | Prix |
+| :-: | :-- |
+| ![Courrone d'Or](/wiki/shared/crown-gold.png "1ère place") | 6 mois d'osu!supporter, un badge de profil unique |
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:27]:
```suggestion
| ![Couronne d'or](/wiki/shared/crown-gold.png "1ère place") | 6 mois d'osu!supporter, un badge de profil unique |
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 30]
```diff
@@ -0,0 +1,117 @@
...
+| ![Courrone d'Argent](/wiki/shared/crown-silver.png "2nde place") | 4 mois d'osu!supporter, un badge de profil unique |
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:28]:
```suggestion
| ![Couronne d'argent](/wiki/shared/crown-silver.png "2nde place") | 4 mois d'osu!supporter, un badge de profil unique |
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 31]
```diff
@@ -0,0 +1,117 @@
...
+| ![Couronne de Bronze](/wiki/shared/crown-bronze.png "3ème place") | 2 mois d'osu!supporter, un badge de profil unique |
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:29]:
```suggestion
| ![Couronne de bronze](/wiki/shared/crown-bronze.png "3ème place") | 2 mois d'osu!supporter, un badge de profil unique |
```

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 32]
```diff
@@ -0,0 +1,117 @@
...
+| ::{ flag=VE }:: [Suminoze](https://osu.ppy.sh/users/5308780) | [1stTry](https://osu.ppy.sh/community/forums/topics/1285565) |
+| ::{ flag=PL }:: [tearek\_](https://osu.ppy.sh/users/20767121) | [Multicolor Diamond HUD](https://osu.ppy.sh/community/forums/topics/1272664) |
+| ::{ flag=VN }:: [TKieen](https://osu.ppy.sh/users/12561202) | [Randoline](https://osu.ppy.sh/community/forums/topics/1291235) |
+| ::{ flag=MX }:: [xet](https://osu.ppy.sh/users/16274977) | [Who Tao](https://osu.ppy.sh/community/forums/topics/1281532) |
+| ::{ flag=PH }:: [Xyber Voltage](https://osu.ppy.sh/users/10443089) | [FreeMod is EZ](https://osu.ppy.sh/community/forums/topics/1269099) |
+| ::{ flag=MX }:: [ZixkyST](https://osu.ppy.sh/users/11844975) | [ZixStar](https://osu.ppy.sh/community/forums/topics/1278132) |
+
+## Podium
+
+*Pour les resultats complets, se référer à la [page du concours](https://osu.ppy.sh/community/contests/126).*
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:96]:
```suggestion
*Pour les résultats complets, se référer à la [page du concours](https://osu.ppy.sh/community/contests/126).*
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Contests/Skinning_Contest/1/fr.md [hunk 33]
```diff
@@ -0,0 +1,117 @@
...
+
+## Prix
+
+| Position | Prix |
+| :-: | :-- |
+| ![Couronne d'or](/wiki/shared/crown-gold.png "1ère place") | 6 mois d'osu!supporter, un badge de profil unique |
+| ![Couronne d'argent](/wiki/shared/crown-silver.png "2nde place") | 4 mois d'osu!supporter, un badge de profil unique |
+| ![Couronne de bronze](/wiki/shared/crown-bronze.png "3ème place") | 2 mois d'osu!supporter, un badge de profil unique |
+
+![](img/badge-winner.png "Badge du gagnant Skinning Contest #1 ") ![](img/badge-2nd.png "Badge de deuxième place du Skinning Contest #1") ![](img/badge-3rd.png "Badge de Troisième place du Skinning Contest #1")
```
Comment (Ganondorf-1) [inline wiki/Contests/Skinning_Contest/1/fr.md:31]:
```suggestion
![](img/badge-winner.png "Badge du gagnant Skinning Contest #1 ") ![](img/badge-2nd.png "Badge de deuxième place du Skinning Contest #1") ![](img/badge-3rd.png "Badge de troisième place du Skinning Contest #1")
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]
(combined 19 commits into 3)

**Pull request merged**